Instructions/09-create-power-bi-dashboard.md

```diff
@@ -87,17 +87,17 @@ In this task, you'll set up the environment for the lab by opening the starter r
 
 In this task, you'll set up the environment for the lab by creating a dataset.
 
-*Important: If you're continuing from the previous lab in the same VM, skip the next task.*
+*Important: If you've already published the dataset, please move to the next task.*
```

Comment:
Just like in Lab 8, 
This assumes that the student knows and completed one of the previous labs. I would just use
If you've already published the dataset in a previous lab, please move to the next task.